roboflow/core/version.py

```diff
@@ -206,19 +205,6 @@ def download(self, model_format=None, location=None, overwrite: bool = False):
 
         self.__wait_if_generating()
 
-        if model_format == "yolov8":
-            # if ultralytics is installed, we will assume users will want to use yolov8 and we check for the supported version  # noqa: E501 // docs
-            try:
-                import_module("ultralytics")
-                print_warn_for_wrong_dependencies_versions([("ultralytics", "==", "8.0.196")])
-            except ImportError:
-                print(
-                    "[WARNING] we noticed you are downloading a `yolov8` datasets but you don't have `ultralytics` installed. "  # noqa: E501 // docs
-                    "Roboflow `.deploy` supports only models trained with `ultralytics==8.0.196`, to intall it `pip install ultralytics==8.0.196`."  # noqa: E501 // docs
-                )
-                # silently fail
-                pass
-
```

Comment:
I'd prefer to remove rather than keep, but let me paint an alternative.

This serves 3 purposes:
* Tell user if the version is too new (e.g. known to not work, but only at this time point)
* Tell user if they're not using ultralytics that we assume they need it
* Tell the maintainer that ultralytics is required.

In the ideal world, we'd instead soften the wording, saying something akin to "When working with yolov8, roboflow works best with ultralytics<=8.2.103. You can install it with ..."

This way, the message is not exactly incorrect in the future, but still gives good info to users.

Leaving it to you to decide what we do. Removal is also fine, in my opinion 🙂

Reply:
mmm idk, IMO this is the wrong place to do this. 
Theres many reasons a user might want to download a version.
It's too weird to be checking ultralytics version as part of downloading a zip file.